samples/web/content/apprtc/apprtc.py

```diff
@@ -230,9 +230,6 @@ def get_room_parameters(request, room_id, client_id, is_initiator):
   video_send_codec = request.get('vsc', default_value = '')
   video_receive_codec = request.get('vrc', default_value = '')
 
-  # Read url param controlling whether we send stereo.
-  stereo = request.get('stereo', default_value = '')
```

Comment:
sorry for being unclear, let's keep audio stereo (this one), but get rid of ssr etc below